Parse/PFObject.h

```diff
@@ -614,7 +614,7 @@ NS_REQUIRES_PROPERTY_DEFINITIONS
 
  @param objects The list of objects to fetch.
  */
-+ (void)fetchAll:(PF_NULLABLE NSArray *)objects;
++ (instancetype)fetchAll:(PF_NULLABLE NSArray *)objects;
```

Comment:
This one should have an `NSArray` return type, since the block has an `NSArray` as a result.